perun-core/src/main/java/cz/metacentrum/perun/core/api/UsersManager.java

```diff
@@ -798,6 +800,18 @@ void changePassword(PerunSession sess, String login, String loginNamespace, Stri
 	void changePassword(PerunSession sess, User user, String loginNamespace, String oldPassword, String newPassword, boolean checkOldPassword)
 			throws PrivilegeException, UserNotExistsException, LoginNotExistsException, PasswordDoesntMatchException, PasswordChangeFailedException, PasswordOperationTimeoutException, PasswordStrengthFailedException, InvalidLoginException, PasswordStrengthException;
 
+	/**
+	 * Checks if the password reset request based on encrypted parameters is valid.
+	 *
+	 * @param sess
+	 * @param i encrypted user id
```

Comment:
Please, if it is not too much work, rename these two variables to some meaningful names in both methods `checkPasswordResetRequestIsValid` and `changeNonAuthzPassword`.

Reply:
They supposed to be meaningless - not to give hints in mails, and also it shortens the URL itself. Generally I would do this in new issue, if you want it. We wanted to check request presence, not to break outer API.

Reply:
ok